packages/openapi-v3/src/json-to-schema.ts

```diff
@@ -94,6 +94,13 @@ export function jsonToSchemaObject(json: JsonSchema): SchemaObject | SchemaRef {
         );
         break;
       }
+      case 'examples': {
+        if (Array.isArray(json.examples)) {
```

Comment:
Should we also check that there is at least one item in the array?

Reply:
If accessing an empty array `json.examples[0]` the result is `undefined`, which is a valid result. This fact is also used by other transformations in `jsonToSchemaObject`.

Reply:
Fair enough 👍